packages/core/src/util/renderer.ts

```diff
@@ -209,7 +217,7 @@ export interface OwnPropsOfControl extends OwnPropsOfRenderer {
 }
 
 export interface OwnPropsOfEnum {
-  options?: any[];
+  options?: EnumOption[];
```

Comment:
This change fails some unit tests as they simply assign strings. They should be adapted.